pystac/asset.py

```diff
@@ -211,3 +213,52 @@ def from_dict(cls: Type[A], d: Dict[str, Any]) -> A:
             roles=roles,
             extra_fields=properties,
         )
+
+    def move(self, href: str) -> Asset:
+        """Moves this asset's file to a new location on the local filesystem,
+        setting the asset href accordingly.
+
+        Modifies the asset in place, and returns the same asset.
```

Comment:
Here, and elsewhere, we should include Google doc styled documentation 

```suggestion
        """Moves this asset's file to a new location on the local filesystem,
        setting the asset href accordingly.

        Modifies the asset in place, and returns the same asset.
        
        Args:
            href: The new asset location
            
        Returns:
            Asset: The asset with the updated href.
```

Reply:
yes yes. Silly me